TeaLeaf/jni/Application.mk

```diff
@@ -1,4 +0,0 @@
-APP_ABI := armeabi-v7a
-APP_PLATFORM := android-14
-APP_STL := gnustl_static
```

Comment:
these too ?

Reply:
Not needed because the project and this file is submodule in AndroidSeed gradle project